influxdb/README.md

```diff
@@ -24,26 +24,26 @@ WARNING:
--	[`1.8-alpine`, `1.8.3-alpine`, `alpine`](https://github.com/influxdata/influxdata-docker/blob/6ffb9c8be705a1fd1672bc9b923fb97ebe3883c3/influxdb/1.8/alpine/Dockerfile)
--	[`1.8-data`, `1.8.3-data`, `data`](https://github.com/influxdata/influxdata-docker/blob/6ffb9c8be705a1fd1672bc9b923fb97ebe3883c3/influxdb/1.8/data/Dockerfile)
--	[`1.8-data-alpine`, `1.8.3-data-alpine`, `data-alpine`](https://github.com/influxdata/influxdata-docker/blob/6ffb9c8be705a1fd1672bc9b923fb97ebe3883c3/influxdb/1.8/data/alpine/Dockerfile)
--	[`1.8-meta`, `1.8.3-meta`, `meta`](https://github.com/influxdata/influxdata-docker/blob/6ffb9c8be705a1fd1672bc9b923fb97ebe3883c3/influxdb/1.8/meta/Dockerfile)
--	[`1.8-meta-alpine`, `1.8.3-meta-alpine`, `meta-alpine`](https://github.com/influxdata/influxdata-docker/blob/6ffb9c8be705a1fd1672bc9b923fb97ebe3883c3/influxdb/1.8/meta/alpine/Dockerfile)
+-	[`1.7`, `1.7.10`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.7/Dockerfile)
+-	[`1.7-alpine`, `1.7.10-alpine`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.7/alpine/Dockerfile)
+-	[`1.7-data`, `1.7.10-data`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.7/data/Dockerfile)
+-	[`1.7-data-alpine`, `1.7.10-data-alpine`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.7/data/alpine/Dockerfile)
+-	[`1.7-meta`, `1.7.10-meta`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.7/meta/Dockerfile)
+-	[`1.7-meta-alpine`, `1.7.10-meta-alpine`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.7/meta/alpine/Dockerfile)
+-	[`1.8`, `1.8.3`, `latest`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.8/Dockerfile)
+-	[`1.8-alpine`, `1.8.3-alpine`, `alpine`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.8/alpine/Dockerfile)
+-	[`1.8-data`, `1.8.3-data`, `data`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.8/data/Dockerfile)
+-	[`1.8-data-alpine`, `1.8.3-data-alpine`, `data-alpine`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.8/data/alpine/Dockerfile)
+-	[`1.8-meta`, `1.8.3-meta`, `meta`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.8/meta/Dockerfile)
+-	[`1.8-meta-alpine`, `1.8.3-meta-alpine`, `meta-alpine`](https://github.com/influxdata/influxdata-docker/blob/6f10af34465b675d9e6de241b57210f01d12b453/influxdb/1.8/meta/alpine/Dockerfile)
```

Comment:
I believe it's expected that `2.0` and variants won't show here until we merge our manifest changes to the official Docker library.